src/sandbox.ts

```diff
@@ -0,0 +1,11 @@
+export function isSandboxed(): boolean {
+  // need to figure out how to inject this during the packaging phase.
+  // see feature flags here: // https://webpack.js.org/plugins/define-plugin/#feature-flags
+  // vsce isn't passing the argument to the vscode:prepublish phase (see package.json)
+  // see this question: https://github.com/microsoft/vscode-vsce/issues/818
+  // consider this post: https://moriz.xyz/blog/feature-flags-in-nextjs
+  if (SANDBOX) {
```

Comment:
this can be webpack build argument

Reply:
Okay.. I saw this is the past you've taken

Reply:
I'm not sure how to make this work in webpack and with vsce . do you know how? You can reference the comments here